Solve kafka deprecated warning

```                                                                                                                                                                                                            
> Task :services:cards-publication-service:compileJava                                                                                                                                                             
/home/didierfre/Projects/opfab/git/operatorfabric-core/services/cards-publication/src/main/java/org/opfab/cards/publication/configuration/kafka/ConsumerFactoryAutoConfiguration.java:48: warning: [removal] buildC
onsumerProperties() in KafkaProperties has been deprecated and marked for removal                                                                                                                                  
        Map<String,Object> props = kafkaProperties.buildConsumerProperties();                                                                                                                                      
                                                  ^                                                                                                                                                                
/home/didierfre/Projects/opfab/git/operatorfabric-core/services/cards-publication/src/main/java/org/opfab/cards/publication/configuration/kafka/ProducerFactoryAutoConfiguration.java:32: warning: [removal] buildP
roducerProperties() in KafkaProperties has been deprecated and marked for removal                                                                                                                                  
        Map<String,Object> props = kafkaProperties.buildProducerProperties();                                                                                                                                      
                                                  ^                                                                                                                                                                
2 warnings                                                                                                                                                                                                         
                                                                                                                                                                                                                   
> Task :services:cards-publication-service:compileTestJava                                                                                                                                                         
/home/didierfre/Projects/opfab/git/operatorfabric-core/services/cards-publication/src/test/java/org/opfab/cards/publication/configuration/kafka/ConsumerFactoryAutoConfigurationShould.java:53: warning: [removal] 
buildConsumerProperties() in KafkaProperties has been deprecated and marked for removal                                                                                                                            
        when(kafkaProperties.buildConsumerProperties()).thenReturn(props);                                                                                                                                         
                            ^                                                                                                                                                                                      
Note: Some input files use unchecked or unsafe operations.                                                                                                                                                         
Note: Recompile with -Xlint:unchecked for details.                                                                                                                                                                 
1 warning                                                                                                                                                                                                          
OpenJDK 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended   


```

--- a/services/cards-publication/src/main/java/org/opfab/cards/publication/configuration/kafka/ConsumerFactoryAutoConfiguration.java
+++ b/services/cards-publication/src/main/java/org/opfab/cards/publication/configuration/kafka/ConsumerFactoryAutoConfiguration.java
@@ -1,5 +1,5 @@
 /* Copyright (c) 2020, Alliander (http://www.alliander.com)
- * Copyright (c) 2021-2023, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021-2024, RTE (http://www.rte-france.com)
  * See AUTHORS.txt
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
@@ -45,7 +45,7 @@ public class ConsumerFactoryAutoConfiguration {
 
     private Map<String,Object> consumerConfig() {
         log.info("bootstrapServers: " + kafkaProperties.getBootstrapServers());
-        Map<String,Object> props = kafkaProperties.buildConsumerProperties();
+        Map<String,Object> props = kafkaProperties.buildConsumerProperties(null);
         props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, ErrorHandlingDeserializer.class);
         props.put(ErrorHandlingDeserializer.KEY_DESERIALIZER_CLASS, deserializerKeyClass);
         props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, ErrorHandlingDeserializer.class);

--- a/services/cards-publication/src/main/java/org/opfab/cards/publication/configuration/kafka/ProducerFactoryAutoConfiguration.java
+++ b/services/cards-publication/src/main/java/org/opfab/cards/publication/configuration/kafka/ProducerFactoryAutoConfiguration.java
@@ -1,3 +1,12 @@
+/* Copyright (c) 2020, Alliander (http://www.alliander.com)
+ * Copyright (c) 2021-2024, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
 package org.opfab.cards.publication.configuration.kafka;
 
 import io.confluent.kafka.serializers.KafkaAvroSerializerConfig;
@@ -29,7 +38,7 @@ public class ProducerFactoryAutoConfiguration {
     private final KafkaProperties kafkaProperties;
 
     private Map<String, Object> producerConfigs() {
-        Map<String,Object> props = kafkaProperties.buildProducerProperties();
+        Map<String,Object> props = kafkaProperties.buildProducerProperties(null);
         props.put(ProducerConfig.CLIENT_ID_CONFIG, "opfab-producer");
         props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
         props.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, valueDeserializer);

--- a/services/cards-publication/src/test/java/org/opfab/cards/publication/configuration/kafka/ConsumerFactoryAutoConfigurationShould.java
+++ b/services/cards-publication/src/test/java/org/opfab/cards/publication/configuration/kafka/ConsumerFactoryAutoConfigurationShould.java
@@ -1,5 +1,5 @@
 /* Copyright (c) 2020, Alliander (http://www.alliander.com)
- * Copyright (c) 2021-2022, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021-2024, RTE (http://www.rte-france.com)
  * See AUTHORS.txt
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
@@ -50,7 +50,7 @@ class ConsumerFactoryAutoConfigurationShould {
     void SetUp() {
         ReflectionTestUtils.setField(cut, "valueDeserializer", "MyValueDeserializer");
         when(kafkaProperties.getBootstrapServers()).thenReturn(new ArrayList<>());
-        when(kafkaProperties.buildConsumerProperties()).thenReturn(props);
+        when(kafkaProperties.buildConsumerProperties(null)).thenReturn(props);
     }
 
     @Test

